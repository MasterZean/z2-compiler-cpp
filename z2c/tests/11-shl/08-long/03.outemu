Long{-2}
Long{-4}
Long{-8}
Long{-16}
Long{-32}
Long{-64}
Long{-128}
Long{-256}
Long{-512}
Long{-1024}
Long{-2048}
Long{-4096}
Long{-8192}
Long{-16384}
Long{-32768}
Long{-65536}
Long{-131072}
Long{-262144}
Long{-524288}
Long{-1048576}
Long{-2097152}
Long{-4194304}
Long{-8388608}
Long{-16777216}
Long{-33554432}
Long{-67108864}
Long{-134217728}
Long{-268435456}
Long{-536870912}
Long{-1073741824}
Long{-2147483648}
Long{-4294967296}
Long{-8589934592}
Long{-17179869184}
Long{-34359738368}
Long{-68719476736}
Long{-137438953472}
Long{-274877906944}
Long{-549755813888}
Long{-1099511627776}
Long{-2199023255552}
Long{-4398046511104}
Long{-8796093022208}
Long{-17592186044416}
Long{-35184372088832}
Long{-70368744177664}
Long{-140737488355328}
Long{-281474976710656}
Long{-562949953421312}
Long{-1125899906842624}
Long{-2251799813685248}
Long{-4503599627370496}
Long{-9007199254740992}
Long{-18014398509481984}
Long{-36028797018963968}
Long{-72057594037927936}
Long{-144115188075855872}
Long{-288230376151711744}
Long{-576460752303423488}
Long{-1152921504606846976}
Long{-2305843009213693952}
Long{-4611686018427387904}
Long{-9223372036854775808}
Long{0}
