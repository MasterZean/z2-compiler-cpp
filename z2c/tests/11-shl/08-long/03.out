-1ll << 1;
-2ll << 1;
-4ll << 1;
-8ll << 1;
-16ll << 1;
-32ll << 1;
-64ll << 1;
-128ll << 1;
-256ll << 1;
-512ll << 1;
-1024ll << 1;
-2048ll << 1;
-4096ll << 1;
-8192ll << 1;
-16384ll << 1;
-32768ll << 1;
-65536ll << 1;
-131072ll << 1;
-262144ll << 1;
-524288ll << 1;
-1048576ll << 1;
-2097152ll << 1;
-4194304ll << 1;
-8388608ll << 1;
-16777216ll << 1;
-33554432ll << 1;
-67108864ll << 1;
-134217728ll << 1;
-268435456ll << 1;
-536870912ll << 1;
-1073741824ll << 1;
-2147483648ll << 1;
-4294967296ll << 1;
-8589934592ll << 1;
-17179869184ll << 1;
-34359738368ll << 1;
-68719476736ll << 1;
-137438953472ll << 1;
-274877906944ll << 1;
-549755813888ll << 1;
-1099511627776ll << 1;
-2199023255552ll << 1;
-4398046511104ll << 1;
-8796093022208ll << 1;
-17592186044416ll << 1;
-35184372088832ll << 1;
-70368744177664ll << 1;
-140737488355328ll << 1;
-281474976710656ll << 1;
-562949953421312ll << 1;
-1125899906842624ll << 1;
-2251799813685248ll << 1;
-4503599627370496ll << 1;
-9007199254740992ll << 1;
-18014398509481984ll << 1;
-36028797018963968ll << 1;
-72057594037927936ll << 1;
-144115188075855872ll << 1;
-288230376151711744ll << 1;
-576460752303423488ll << 1;
-1152921504606846976ll << 1;
-2305843009213693952ll << 1;
-4611686018427387904ll << 1;
-9223372036854775808ll << 1;
